src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -75,40 +75,43 @@ export class GraphStyles {
     const getEdgeLabel = (ele: any): string => {
       const cyGlobal = getCyGlobalData(ele);
       const edgeLabelMode = cyGlobal.edgeLabelMode;
+      let content = '';
 
       switch (edgeLabelMode) {
-        case EdgeLabelMode.REQUESTS_PER_SECOND: {
-          const rate = ele.data('rate') ? parseFloat(ele.data('rate')) : 0;
-          if (rate > 0) {
-            const pErr = ele.data('percentErr') ? parseFloat(ele.data('percentErr')) : 0;
-            return pErr > 0 ? rate.toFixed(2) + ', ' + pErr.toFixed(1) + '%' : rate.toFixed(2);
+        case EdgeLabelMode.TRAFFIC_RATE_PER_SECOND: {
+          if (ele.data('rate')) {
+            const rate = Number(ele.data('rate'));
+            if (rate > 0) {
+              const pErr = ele.data('percentErr') ? Number(ele.data('percentErr')) : 0;
+              content = pErr > 0 ? rate.toFixed(2) + ', ' + pErr.toFixed(1) + '%' : rate.toFixed(2);
+            }
+          } else if (ele.data('tcpSentRate')) {
+            const rate = Number(ele.data('tcpSentRate'));
+            content = `${rate.toFixed(2)}`;
```

Comment:
looking at the code this refactored, did you miss a formatting thing? Down below, this was removed

```
const tcpRate = ele.data('tcpSentRate') ? parseFloat(ele.data('tcpSentRate')) : 0;
if (tcpRate > 0) {
  return `${tcpRate.toFixed(2)}`;
}
```
Notice if tcpRate is > 0 then its fixed to 2 decimal places (toFixed(2)). But it specifically doesn't do this if its 0. Here, you do it regardless of zero or non-zero. I wonder if this means you will see "0.00" now instead of just "0"?

Reply:
Yes, i think I missed on something here.  Will fix.

Reply:
No, looking again, I think it's OK as is.  if you look at the old code for TCP rate, is tcpRate is 0 we just return '', because we don't want to actually see any 0 values on the edges.

Reply:
We never send a 0 from the backend.  But I will add the check anyway.